front/public/locales/fr/errors.json

```
@@ -238,8 +238,9 @@
     },
     "paced_train": {
       "Database": "Erreur interne (base de données)",
-      "BatchPacedTrainNotFound": "Certaines missions sont introuvables",
-      "PacedTrainNotFound": "Mission '{{paced_train_id}}' non trouvée"
+      "BatchNotFound": "'{{count}}' missions sont introuvables",
```

```suggestion
      "BatchNotFound": "'{{count}}' mission(s) sont introuvable(s)",
```

nit++, but I really dislike seeing things like "1 objets créés"